src/agent/src/random.rs

```diff
@@ -44,3 +44,24 @@ pub fn reseed_rng(data: &[u8]) -> Result<()> {
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use crate::skip_if_not_root;
+    use std::fs::File;
+    use std::io::prelude::*;
+
+    #[test]
+    fn test_reseed_rng() {
+        skip_if_not_root!();
+        const POOL_SIZE: usize = 512;
+        let mut f = File::open("/dev/random").unwrap();
```

Comment:
`/dev/random` can block. We just need a seed somewhere. Please either read from `urandom` or from any file locally.

Reply:
@bergwolf does it work for you if I have the seed hardcoded on the code file?

Reply:
Maybe what I said is not a good idea. I will think in how to read from file or maybe just use `urandom`

Reply:
done